core/src/main/java/hudson/model/DirectoryBrowserSupport.java

```diff
@@ -404,12 +404,18 @@ private static void zip(OutputStream outputStream, VirtualFile dir, String glob)
          */
         private final boolean isReadable;
 
-        public Path(String href, String title, boolean isFolder, long size, boolean isReadable) {
+       /**
+        * For a file, the last modified timestamp.
+        */
+        private final long lastModified;
```

Comment:
`0L` requires special processing in UI. Otherwise it will show a weird date.

Reply:
getLastModifiedAsCalendar make the transformation to Canlendar for use in view, if it is 0L the show date is 00:00:00 GMT, January 1, 1970.